src/constants/general.js

```diff
@@ -18,6 +18,7 @@ export default {
 
     MENTION: 'mention',
 
+    OUT_OF_OFFICE: 'ooo',
```

Comment:
I don't love the `ooo` value, why not `out_of_office`?

Reply:
+1 for ooo 😄

Reply:
@jespino I followed the convention with DND.  `dnd`.  From my assumption, it seemed like the reasoning for `do_not_disturb` felt inadequate?

Therefore I went with `ooo`.

Also, from a feature standpoint, slash commands follow the same format /dnd, therefore once slash command for Out Of Office is implemented we would use `/ooo`

Reply:
ok, makes sense :)